### DISCUSSION TITLE
Ignition wood surface

### DISCUSSION THREAD

**Discussion by nathanremans** [firemodels/fds#12358] [General]
Hi everyone! 

In the attached file I try to model the ignition of a wood surface. However, I set the reference temperature of the reaction to 35 degrees (as a test, not a realistic value), and the ambient temperature to 20 degrees. It would be expected that the wood surface does not ignite, but it does show a burning rate and flaming. Can anyone explain what causes ignition if the ambient temperature should be too low for the surface to ignite?

Kind regards,
Nathan 

[model_setup_testvolume.txt](https://github.com/firemodels/fds/files/13902705/model_setup_testvolume.txt)

**Comment by drjfloyd** [comment]
The reference temperature is not an ignition temperature. The reference temperature is where the peak rate occurs.  Pyrolysis will start at a lower temperature.

**Reply by nathanremans** [reply]
Thank you! Would you suggest setting the ignition temperature then? For wood pyrolysis in particular, it usually happens in between 250 and 450 degrees. Because right now it does not seem rational to have a model which gives a HRR coming from the walls at ambient conditions.

**Comment by drjfloyd** [comment]
You have a MATL definition with a non-physical REFERENCE_TEMPERATURE. With your inputs pyrolysis is going to start in earnest around 0 C and be essentially done by around 60 C. If you want the wood to start burning at 250 C or more you need a reference temperature above that
